packages/design-system/src/components/IdleTimeout/useInterval.js

```diff
@@ -0,0 +1,24 @@
+import { useEffect, useRef } from 'react';
+
+// https://overreacted.io/making-setinterval-declarative-with-react-hooks/
+function useInterval(callback, delay) {
```

Comment:
Nice 👌 . I actually missed that you were adding this yourself the first time. It's a nice hook!